ui/module.go

```diff
@@ -27,6 +29,11 @@ func prep() error {
 }
 
 func start() error {
+	err := dataroot.Root().ChildDir("exec", 0777).Ensure()
```

Comment:
This is just a dummy directory to which processes change their working directory to. Currently this includes the App and the Notifier. The aim is protect all other directories and increase compatibility should any process want to read or write something to the current working directory. This can also be useful in the future to dump data to for debugging.

Reply:
yeah but I'd go with `0700` because if you ever read data from that directory only the user should be allowed to modify/write to it and not everyone.

Reply:
Resolved in call. Added a note.